src/sidebar/components/annotation.js

```diff
@@ -117,15 +117,15 @@ function Annotation({
       {isEditing && <TagEditor onEditTags={onEditTags} tagList={tags} />}
       {!isEditing && <TagList annotation={annotation} tags={tags} />}
       <footer className="annotation__footer">
-        <div className="annotation__form-actions">
-          {isEditing && (
+        {isEditing && (
```

Comment:
This seemingly unrelated change is because I realized that rendering an empty `.annotation__form-actions` `div` is part of our vertical spacing problem with annotation cards. ONE PART. MANY OTHER PARTS.